package.json

```diff
@@ -1,20 +1,59 @@
 {
   "name": "css-frameworks-ca",
   "version": "1.0.0",
-  "description": "Test",
-  "main": "index.js",
+  "description": "social media test",
+  "type": "module",
+  "main": "src/js/index.js",
   "scripts": {
+    "test": "npm run test-unit",
+    "test-unit": "jest",
+    "test-cypress": "npm run test-e2e",
+    "test-e2e": "cypress open",
+    "test-e2e-cli": "cypress run",
+    "localhost-cypress": "start-server-and-test start http://127.0.0.1:8080 test-cypress",
```

Comment:
Cypress comes with its own server, so you can run your tests against `/` and they will work!

Reply:
The redundant lines (13 & 14) are removed